services/api-db/docker-entrypoint-initdb.d/01-migrations.sql

```diff
@@ -645,6 +645,25 @@ CREATE OR REPLACE PROCEDURE
   END;
 $$
 
+CREATE OR REPLACE PROCEDURE
+  update_env_vars_from_varchar_to_text()
```

Comment:
let's use `convert_` instead of `update_` as other similar procedure are already using `convert`: `convert_task_command_to_text`

Reply:
Changed.